pkg/config/log.go

```diff
@@ -51,23 +51,6 @@ func createQuoteMsgFormatter(params string) seelog.FormatterFunc {
 	}
 }
 
-// buildCommonFormat returns the log common format seelog string
-func buildCommonFormat(loggerName LoggerName) string {
-	if loggerName == "JMXFETCH" {
-		return `%Msg%n`
-	}
```

Comment:
I think this should be included in `pkg/config/log_format.go`, for both `buildCommonFormat` and `buildJSONFormat`.

Reply:
Very nice catch, this must be a merge error or merge artifact.